apartment-app/apartment-app-intro.md

```diff
@@ -0,0 +1,129 @@
+# Apartment App
+Apartment app is a full stack application that we will be building in small groups, essentially as dev teams.  In this project, you will build the application, work through CRUD actions on both front and back end, test your work using React testing library and Rspec, style the components as you work through them, and work together as a team to complete this task.  We will be utilizing Trello as a project management software to help keep track of what is in progress and the tasks that are needed to be completed.  
+
+Since we will be working in groups of fours, there will be a little different dynamic when it comes to our project management.  Each team will be responsible for communication and be in agreement with each other on who is working on what tasks.  While you it is up to you on how you want to work in your teams, many groups find that splitting off into two pairs is helpful, just make sure each pair is still communicating with the other half of the team on blockers or progress made.  
+
+As instructors, we will be around to help wherever and whenever possible and will also be reviewing PR's as they come through.  We will periodically check on your Trello board and will ask questions if something seems out of place.
+
+We are going to go through a few of the initial steps together, starting with our Trello cards.
+
+## Trello
+We used Trello with Cat Tinder, where you were provided cards with specific tasks.  This time, together we are going to collectively think about the content that we are going to need as a class, and prioritize these items accordingly.
+
+(Each first dash - is a trello cards, the following dashes are checklists inside the trello card)  
+
+- Initial Setup (Located in apartment-app section in syllabus)
+    - Installs
+    - Devise
+    - React in Rails
+    - Reactstrap
+    - Add jest
+
+- Application Structure    
+    - Apartment Resource
+        - Associations (User has_many Apartments, Apartment belongs_to User)
```

Comment:
Should this include model validations?

Reply:
Cool, I am adding it the request spec for 422 error onto the create and update cards and the model specs to apartment structure.